proposal/yanagiragi/readme.md

```diff
@@ -0,0 +1,147 @@
+# Proposal for TinySandbox
+
+## Basic information
+
+---
+
+Github Repo: [Tiny Sandbox](https://github.com/yanagiragi/TinySandbox)
+
+I will try to build a simple game engine from scratch. The simple engine is proposed to be a sandbox for learning and trying new techniques.
+
+
+## Problem to solve
+
+---
+
+Quick prototyping is always a pain in the ass. 
```

Comment:
Quantitative descriptions will be more illustrative than the qualitative ones you are using here, but I get your points.